test/test-functions

```diff
@@ -750,7 +750,7 @@ install_compiled_systemd() {
 install_debian_systemd() {
     ddebug "Install debian systemd"
 
-    local _systemd_pkgs=$(apt-cache showsrc systemd | grep -m 1 -E '^Binary:' | cut -d ':' -f 2 | tr -d ,)
+    local _systemd_pkgs=$(grep -E '^Package:' ${SOURCE_DIR}/debian/control | cut -d ':' -f 2 | tr '\n' ' ')
```

Comment:
That last `tr` isn't necessary.

Reply:
the previous command printed a single line - I'll check that it works the same with multiple lines

Reply:
Yep it's not needed, removed